massa-node/base_config/openrpc.json

```diff
@@ -1873,8 +1888,7 @@
                         "$ref": "#/components/schemas/ExecutedAt"
                     },
                     "result": {
-                        "description": "\"ok\" or error message",
-                        "type": "string"
+                        "$ref": "#/components/schemas/ReadOnlyResult"
```

Comment:
The structure is more complex than a `Vec<u8>` because it include error message in case of error. Maybe it's not the most idiomatic way in JSON to format data but the structure [existed before](https://github.com/massalabs/massa/pull/3339/files#diff-76fc34f42f93b2ee554255e7f9ef7caa64a4abe3f3ce5a8d09e768d9418e2ab4R8) and I fixed the doc and added my field. 

The doc was wrong as we can this on [this playground]( https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=38131a27705f89e0ba8673790fe1f7d2) in case of `Ok` it returned the right string but in case of `Error` it return an object which wasn't specified in the doc.